dlc_developer_config.toml

```diff
@@ -23,7 +23,7 @@ benchmark_mode = false
 [build]
```

Comment:
enable `benchmark_mode` in line 21. We are adding performance benchmarks.

Reply:
I turned on benchmark_mode.  I know we still have @BruceChang9688's changes incoming, but at least we can get a baseline test result from the existing benchmark tests.